### ISSUE TITLE
Problem na produkcyjnym Allegro z atrybutem

### ISSUE THREAD

**Issue by mateuszcom** [allegro/allegro-api#5835]
Wystawiając produkt w kategorii Dom i Ogród - Oświetlenie - Lampy - Lampki biurkowe [259810] (produkcyjne Allegro), po wybraniu atrybutu KOLOR -> INNY integracja wymaga od nas przekazania wartości niejednoznacznej. Wystawiając przez formularz, nie mamy takiego obowiązku. Do tej pory spotkaliśmy się tylko z wartościami niejednoznacznymi dla parametrów z listy rozwijanej, nie dla checkboxów. Nie wiem, czy to błąd, czy mamy uwzględnić wartości niejednoznaczne również dla parametrów w checkboxach

**Comment by PrzemyslawLukanowski** [comment]
Czy mógłbyś podać trace-id żądania, gdzie otrzymaliście taki komunikat dla API? Parametr ten oznaczony jest jako `"customValuesEnabled": false`, dlatego nie powinno być w nim możliwości przekazania własnej wartości.

**Comment by PrzemyslawLukanowski** [comment]
Dzięki trace-id możemy namierzyć Twój request oraz odpowiedź naszego API, powinieneś go otrzymać w nagłówkach odpowiedzi. W obecnej sytuacji nie będzie to potrzebne, opis błędu, który wysłałeś, nam wystarczy. 
Komunikat błędu nie wskazuje na to, że wymagamy, abyś podał wartość niejednoznaczną, ale na to, że ta wartość jest nieprawidłowa w kontekście tworzenia produktu. W tym przypadku musisz przekazać jedną z dostępnych opcji, ale nie niejednoznaczną.

**Comment by mateuszcom** [comment]
Nie do końca rozumiem. W takim razie wybranie opcji "INNY" pomimo istnienia takiej opcji wyboru jest błędne? Po wybraniu jakiegokolwiek koloru (z pominięciem INNY) powoduje poprane utworzenie produktu. Co w przypadku gdy pożądanego koloru nie ma w liście i musimy wybrać INNY?

**Comment by PrzemyslawLukanowski** [comment]
> W takim razie wybranie opcji "INNY" pomimo istnienia takiej opcji wyboru jest błędne? 

Tak, w przypadku, gdy tworzysz nowy produkt, musisz wskazać konkretną wartość - nie niejednoznaczną. 

> Co w przypadku gdy pożądanego koloru nie ma w liście i musimy wybrać INNY?

W takiej sytuacji możesz zaproponować wraz z argumentacją nową wartość przez nasz [formularz kontaktowy.](https://allegro.pl/pomoc/kontakt)
